src/components/header/modals/ModalConnectWallet.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <astar-simple-modal :show="isModalConnectWallet" title="Select a Wallet" @close="setCloseModal">
+  <ModalDrawer :show="isModalConnectWallet" title="Select a Wallet" @close="setCloseModal">
```

Comment:
![Screen Shot 2022-04-11 at 9 44 45 AM](https://user-images.githubusercontent.com/40356749/162688958-08f3625f-ace1-40c8-8d05-09775a8f5acf.png)
![Screen Shot 2022-04-11 at 9 44 51 AM](https://user-images.githubusercontent.com/40356749/162688964-dd15af3f-b72a-4220-82b1-398170b10ba3.png)

The modal can be scrolled. Please adjust it so that the bottom button is always visible and we only have only scrollable view

Reply:
what's the maximum scrollable height for the account list? in current portal, `connect button` is not always visible as well. This requires a lot of scrolling down when there are many accounts. please guide me for this @Kahonnohak

Reply:
You should be able to see three accounts without scrolling. Do many users have more than 10+ accounts? Toga said he does, in that case, design might need to change  it probably will be an accordion) I'll ask people around. But we anyway need a scroller, please check the design.
<img width="284" alt="スクリーンショット 2022-04-12 14 13 58" src="https://user-images.githubusercontent.com/77480847/162885603-3072396a-ed39-451f-934f-5e36bc056ff5.png">

Let me comeback for the review in a couple of hours time! sorry for being slow.

Reply:
Yes the button needs to be visible. I thought I would fix the height but how about we make the modal height the same as the browser height. We do need to be able to scroll though, can we not?
<img width="964" alt="スクリーンショット 2022-04-12 17 07 57" src="https://user-images.githubusercontent.com/77480847/162912585-2b07302a-1acd-4ce8-be36-b72b62222dc9.png">

Reply:
@sirius651 Thanks! Yes better:) I got few points I would like you to adjust. sorry but let me come back in an hour.

Reply:
<img width="334" alt="スクリーンショット 2022-04-13 12 11 56" src="https://user-images.githubusercontent.com/77480847/163092647-d7314414-7696-40ed-8cfa-e7422147f35b.png">
For the polkadot wallet

1) Please remove the round corner please (both top corners)

2) Could you remove this drop shadow please? It looks like there is another subtle one which I would like to keep.
.modal[data-v-46c69d34]
<img width="334" alt="スクリーンショット 2022-04-13 12 17 37" src="https://user-images.githubusercontent.com/77480847/163093358-33608eec-b131-42b7-b42f-9fa3d3694954.png">

3) Would you work on the radio button please? (This is probably UI lib)

4) Could we add "connect" mark for the account that is currently connected?

5) 8px between accounts

6) 40px above the big button

7) Could you add slide animation please.
https://animate.style/
animate__slideInRight

<img width="334" alt="スクリーンショット 2022-04-13 12 11 56" src="https://user-images.githubusercontent.com/77480847/163094169-284e8533-c5b8-42a9-b52e-701ec53c0854.png">

Thanks

Reply:
![스크린샷 2022-04-14 오전 11 58 56](https://user-images.githubusercontent.com/1606820/163305855-ce9bb151-b669-4284-bd4e-7c3e63cb8912.png)
I made changed above. could you check again?